File: docs/src/testers/sessions/cdp/selenium_builtin_page.ts
// @sniptest filename=selenium_builtin_page.ts
// @sniptest show=1-8
import { NotteClient } from "notte-sdk";

const client = new NotteClient();

const session = client.Session();
await session.use(async (session) => {
  const page = await session.page(); // Built-in Playwright page
});

const status = await session.status();
export { status };

File: docs/src/testers/sessions/cdp/selenium_builtin_page.py
# @sniptest filename=selenium_builtin_page.py
# @sniptest show=1-6
from notte_sdk import NotteClient

client = NotteClient()

with client.Session() as session:
    page = session.page  # Built-in Playwright page
    assert page.evaluate("1 + 1") == 2
    status = session.status()
